data/authors.yml

```
@@ -31,3 +31,6 @@
 - name: Tim Quirino
   avatar: /authors/profile-tim-quirino.png
   title: Staff Product Designer
+- name: Jacob Ebey
+  avatar: /authors/jacob-ebey.jpg
+  title: Senior Developer
```

I mean maybe Tim had a bigger impact on RSC than I realized, but I think it should be:

```suggestion
- name: Jacob Ebey
  avatar: /authors/jacob-ebey.jpg
  title: Senior Developer
```

---

I'm dumb, this is the `authors.yml` file 🙃